arrows/vxl/image_io.cxx

```diff
@@ -18,10 +18,14 @@
 #include <arrows/vxl/image_container.h>
 
 #include <vil/vil_convert.h>
+#include <vil/vil_plane.h>
```

Comment:
I would recommend squashing the changes to this one file into a single commit, add test case for reading and writing split images, and make that a new branch for just that change.  That should be easy to get merged by itself.